DEVELOPMENT.md

```diff
@@ -9,21 +9,38 @@ The Pixie project currently consists of three main components:
 This document outlines the process for setting up the development environment for each of these components.
 
 ## Setting up the Environment
+There are two different ways to run a development environment. The first and easiest approach is via Docker. If you plan to use a non-Minikube environment (GKE, EKS, etc) for running Vizier/Pixie Cloud, use this first approach via Docker. The second approach is using Make and Minikube.
```

Comment:
This will change because we are removing references to the make way, right? Also, I don't think there is a GKE/EKS/Minikube association either way with either approach

Reply:
I've deleted the second method. This is all inside a single commit so if we need to role it back we can.